wrappers/rust/icicle-core/src/merkle/mod.rs

```diff
@@ -263,11 +265,14 @@ impl MerkleTree {
         }
 
         let proof = MerkleProof::new().unwrap();
+        let byte_size = (leaves.len() * std::mem::size_of::<T>()) as u64;
         let result = unsafe {
             icicle_merkle_tree_get_proof(
                 self.handle,
                 leaves.as_ptr() as *const u8,
+                byte_size,
                 leaf_idx as u64,
+                false,
```

Comment:
why false? you need to take this argument

Reply:
you wrote this file, no?

Reply:
Yes but Koren added this argument. I will fix it